### ISSUE TITLE
'H' key doesn't work as second character when creating new folder / file (on atom keymap)

### ISSUE THREAD

**Issue by arjuncode** [zed-industries/community#1100]
### Check for existing issues

- [X] Completed

### Describe the bug / provide steps to reproduce it

I tried to create a new folder/file with a name like 'chart/' or 'theme.py', but the 'H' key doesn't seem to work when it is used as the second character in a new folder/file name. No other character seems to be blocked from being used as the second character in the new folder/file name. It is worth noting that the 'H' key works for the first character in a folder/file name, like 'home/'.

### Environment

Zed: v0.77.2 (stable)
OS: macOS 12.0.0
Memory: 32 GiB
Architecture: aarch64

### If applicable, attach your `~/Library/Logs/Zed/Zed.log` file to this issue.
If you only need the most recent lines, you can run the `zed: open log` command palette action to see the last 1000.

_No response_

**Comment by jrasanen** [comment]
Worked for me. Perhaps logs would reveal more about what went wrong on your device?

https://user-images.githubusercontent.com/13303/225236701-3cf3b171-87e0-4ff6-bca2-1a3e1257304b.mp4

**Comment by JosephTLyons** [comment]
This is super strange.  I'm unable to reproduce this, wonder what the difference might be.  Do you have `vim_mode` enabled by chance?

**Comment by JosephTLyons** [comment]
Hey @arjuncode, by chance, are you using the Atom keymap with Zed?  It looks like we have a bug with Zed where it consume key combos, even when editing the file name.  In the Atom keymap, there is an action associated with just the letter `h`.  

```json
  {
    "context": "ProjectPanel",
    "bindings": {
      "h": "project_panel::CollapseSelectedEntry",
    }
  },
```

Zed is interpreting `h` as the user wanting to launch that action, even while typing.  As a short term patch,I updated the atom keymap to use a different combo (Atom actually assigns 3 different key combos to this action! so I just picked a different one).  One issue though is that our base keymaps are currently baked into Zed, so users can't just edit them - meaning you will either need to use a different keymap until this lands in stable, or bear with it (maybe copy and paste the text you want for your file name).

I'm going to talk to the team about getting the actual bug fixed, as well what we should do to improve the ease of users bringing in updated keymaps whenever they want.

**Comment by JosephTLyons** [comment]
Additionally, in the meanwhile, you could:

1. copy the contents of the Atom base keymap
2. paste them over your custom keymap file (merging in whatever else you need)
3. Swap the `h` keybinding out for the `opt-b` one.

**Comment by arjuncode** [comment]
> Hey @arjuncode, by chance, are you using the Atom keymap with Zed? It looks like we have a bug with Zed where it consume key combos, even when editing the file name. In the Atom keymap, there is an action associated with just the letter `h`.
> 
> ```json
>   {
>     "context": "ProjectPanel",
>     "bindings": {
>       "h": "project_panel::CollapseSelectedEntry",
>     }
>   },
> ```
> 
> Zed is interpreting `h` as the user wanting to launch that action, even while typing. As a short term patch,I updated the atom keymap to use a different combo (Atom actually assigns 3 different key combos to this action! so I just picked a different one). One issue though is that our base keymaps are currently baked into Zed, so users can't just edit them - meaning you will either need to use a different keymap until this lands in stable, or bear with it (maybe copy and paste the text you want for your file name).
> 
> I'm going to talk to the team about getting the actual bug fixed, as well what we should do to improve the ease of users bringing in updated keymaps whenever they want.

Hi! This is it! I _am_ on atom keymap. Thank you for the workaround. I'll leave this open for the team to close when they see it appropriate.

**Comment by arjuncode** [comment]
Edited title to ease future user issue discovery

**Comment by JosephTLyons** [comment]
The patch for this should be landing in our next stable release (likely tomorrow).  This patch is just changing the atom keymap keybinding for that action and not a long-term fix with the project panel consuming keybindings when editing a file name.

**Comment by JosephTLyons** [comment]
This should be fixed in v0.78.x.  Please reopen if the issue is still present after the update.
